MAUI/Chips/Customization.md

```diff
@@ -1047,6 +1047,48 @@ public MainPage()
 
 ![SfChipGroup with SelectedChipBackgroundColor](images/customization-images/chipgroup_chipbackground_image.png)
 
+## SelectedChipBackground
+
+The `SelectedChipBackground` property customizes the background color of the selected chip.
```

Comment:
The `SelectedChipBackground` property customizes the background color of the selected chips.

Reply:
Changed